nimble/host/syscfg.yml

```diff
@@ -523,6 +514,20 @@ syscfg.defs:
         description: 'Minimum level for the BLE EATT log.'
         value: 1
 
+    BLE_MAX_BIG:
```

Comment:
BLE_ISO_MAX_BIGS or BLE_AUDIO_MAX_BIGS

I think the former makes more sense

Reply:
I left ISO in host, so I'll change to that

Reply:
And BLE_ISO_MAX_BISES